### DISCUSSION TITLE
Improvements from our documentation employees

### DISCUSSION THREAD

**Discussion by designatedsuccessor** [outline/outline#7191] [Ideas]
Our documentation employees have been using Outline for a few weeks now.  Here is some preliminary feedback on what they say would make their lives easier in Outline:

- Tables: allow optional hiding of all borders.
- Tables: allow optional alternating background colors in rows (e.g., first row is a light gray background, second row is a darker gray background, third row is back to the light gray background, etc.).
- Tables: allow merging cells.
- Tool tips: currently, most hyperlinks link to other web pages. As an addition, add a feature that makes it where a hyperlink can link to "tool tip" that allows the reader to get a small amount of info without going to another webpage, by making a small info box appear.  Example where clicking "Intro to Computers" makes a foreground box appear with more info about the "Intro to Computers" class:
![image](https://github.com/outline/outline/assets/25268183/9cf951c0-1fa3-48d5-9cfb-f8f71fe7a43a)
In our environment, it would be preferable to have a separate "snippets" area in the interface from which these could be pulled, since some "snippets" may be used many times throughout our documentation pages.  Example: we might have an "Intro to Computers" link on several pages throughout our Outline instance, and we want to pull from the same "snippet" every time.

**Comment by tommoor** [comment]
Thanks for the feedback

- Merging table cells is tracked here: #6514 
- Tooltips exists as a feature, if you're self hosting you would need to enable the [Iframely Integration](https://docs.getoutline.com/s/hosting/doc/iframely-HwLF1EZ9mo) to enable fetching previews of third party sites!

**Reply by designatedsuccessor** [reply]
We are cloud hosted.  Does that mean we can't implement?

**Reply by tommoor** [reply]
It's enabled by default in cloud, hover over any link to see a preview

<img width="504" alt="image" src="https://github.com/outline/outline/assets/380914/7b1d102e-d0ca-4446-b581-cce1b3da20c2">

**Reply by designatedsuccessor** [reply]
Sorry, I don't think I'm being clear about my request.  When I say "tool tip," I don't really mean "web preview."  I mean a mini-page of info.

Put another way: I'm talking about using `<a href="">` elements that don't link to other pages at all but instead show whatever info you want it to show.

In the example image I showed in the OP, the "Intro to Computers" link does not link to another page at all.  When you click it, it makes a foreground box appear with info inside it that gives you some basic info about the "Intro to Computers" class.  If you click the link again, the box disappears.

This would helpful when you want the user to, for example, get some basic understanding of a term or phrase without having to entirely leaving the page.

**Reply by tommoor** [reply]
I see, thanks for the explanation!
